build-aux/Jenkinsfile.full

```diff
@@ -47,6 +47,12 @@ meta = [
     image: "apache/couchdbci-ubuntu:focal-erlang-${ERLANG_VERSION}"
   ],
 
+  'jammy': [
+      name: 'Ubuntu 22.04',
+      spidermonkey_vsn: '91',
+      image: "apache/couchdbci-ubuntu:jammy-erlang-${ERLANG_VERSION}"
```

Comment:
I pushed it in https://github.com/apache/couchdb-ci/pull/38 as a `apache/couchdbci-ubuntu:jammy-erlang-default` default there is the erlang 24 version. 

If it fails with pidermonkey_vsn: '91', can try with `78` as it has both of them available in the apt repo.